src/TransitionDispatchers/SymfonyDispatcher.php

```diff
@@ -25,9 +26,9 @@ public function __construct(EventDispatcherInterface $eventDispatcher)
     /**
      * {@inheritdoc}
      */
-    public function dispatch(CircuitBreaker $circuitBreaker, $transition, $service, array $parameters): void
+    public function dispatch(CircuitBreaker $circuitBreaker, Service $service, string $transition): void
     {
-        $event = new TransitionEvent($circuitBreaker, $transition, $service, $parameters);
+        $event = new TransitionEvent($circuitBreaker, $transition, $service);
```

Comment:
future improvement: 1 transition event class by place so we could change the API to dispatch(TransitionEventInterface $event)